submissions/andrewklmn/html-css-cinema-booking/index.html

```diff
@@ -0,0 +1,501 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ticket booking form</title>
+  </head>
+  <body data-new-gr-c-s-check-loaded="8.869.0" data-gr-ext-installed="">
+    <header aria-label="Page header with navigation">
+      <nav aria-label="Bavigation bar">
+        <a href=""><img class="logo" src="img/logo.png" alt="Logo of cinema"></a>
+        <ol class="menu" aria-label="Set of links">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contacts</a></li>
+        </ol>
+      </nav>
+    </header>
+    <main aria-label="Main area of booking form">
+      <section class="poster" aria-label="Poster about movie">
+        <img class="poster_image" 
+              srcset="img/banner-horizontal_480w.jpg 480w,
+                      img/banner-horizontal_992w.jpg 992w"
+              sizes="(max-width: 500px) 480px, 992px"
+              src="img/banner-horizontal.jpg"
+              alt="Poster of selected movie">
+        <h1>Interstellar (2014)</h1>
+        <p class="about">
+          In Earth's future, a global crop blight and second Dust Bowl are slowly 
+          rendering the planet uninhabitable. Professor Brand (Michael Caine), 
+          a brilliant NASA physicist, is working on plans to save mankind by 
+          transporting Earth's population to a new home via a wormhole. 
+          But first, Brand must send former NASA pilot Cooper (Matthew McConaughey) 
+          and a team of researchers through the wormhole and across the galaxy 
+          to find out which of three planets could be mankind's new home. 
+        </p>
+        <hr>
+        <section class="crew">
+          <h1 class="hidden">Crew</h1>
+          <p><strong>Genre:</strong> sci fi, action, adventure</p>
+          <p><strong>Original Language:</strong> English</p>
+          <p><strong>Director:</strong> Christopher Nolan </p>
+          <p><strong>Director:</strong> Jonathan Nolan, Christopher Nolan</p>
+          <p><strong>Cast:</strong> Matthew McConaughey, Anne Hathaway, Jessica Chastain and others</p>
+        </section> 
+      </section>
+      <section class="interactive" aria-label="Area for setting of booking details">
+        <h1 class="hidden">Ticket booking form</h1>      
+        <form class="booking_form" action="index.html" method="POST" autocomplete="off" aria-label="Interactive area">
+          <section class="date_selector" aria-label="Date selector">
+            <h1>Available dates:</h1>
+            <input class="checkbox_control disappeared" type="radio" name="show_date" id="show_date1" value="08 Feb" aria-label="8th February" aria-checked="true" checked="checked">
+            <label class="checkbox_label" for="show_date1" aria-hidden="true">08 Feb</label>
```

Comment:
For date use tag `time` with attribute `datetime`. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)